source/opt/strip_debug_info_pass.cpp

```diff
@@ -22,10 +22,19 @@ Pass::Status StripDebugInfoPass::Process() {
   bool modified = !context()->debugs1().empty() ||
                   !context()->debugs2().empty() ||
                   !context()->debugs3().empty();
+  for (auto& dbg : context()->debugs1()) context()->KillNamesAndDecorates(&dbg);
+
+  for (auto& dbg : context()->debugs2()) context()->KillNamesAndDecorates(&dbg);
+
+  for (auto& dbg : context()->debugs3()) context()->KillNamesAndDecorates(&dbg);
+
   context()->debug_clear();
 
-  context()->module()->ForEachInst([&modified](Instruction* inst) {
+  context()->module()->ForEachInst([&modified, this](Instruction* inst) {
     modified |= !inst->dbg_line_insts().empty();
+    for (auto& dbg : inst->dbg_line_insts())
+      this->context()->KillNamesAndDecorates(&dbg);
+
     inst->dbg_line_insts().clear();
```

Comment:
OpLine instruction are different.  Many of the analysies do not even consider the OpLine instructions.  For now, we can leave this clear.  Eventually we may change the way we handle debug info, then everything will have to change.

Reply:
Done